app/assets/stylesheets/main.scss

```diff
@@ -3,6 +3,8 @@
 @font-face {
   font-family: 'FinkHeavy';    
   src: asset-path(FinkHeavy) format("opentype");
+  src: font_url('FinkHeavy.otf'); 
+  src: font-url('FinkHeavy.otf'); 
```

Comment:
duplication here!

Reply:
That was intentional! One is a hypen "-" and the other is an underscore "_" - apparently Heroku gets weird about the formatting so I'm including both!